bin/hbase

```diff
@@ -767,6 +772,11 @@ elif [ "${DEBUG}" = "true" ]; then
   echo "JDK11 jars skipped from classpath."
 fi
 
+if [[ -n "${HBASE_TRACE_OPTS}" ]]; then
+  echo "Attach opentelemetry agent to enable trace"
+  enable_trace
```

Comment:
This trick looks good. Where will you document it? Maybe if there was an 'hbase trace' command and if you ran it, you'd get output telling you to populate HBASE_TRACE_OPTs giving examples... perhaps that would be enough to get folks going? Would also advertise the new facility exits.

Reply:
In hbase-env.sh, uncomment the last line and modify the options. Will fill this in the ref guide.